include/package-defaults.mk

```diff
@@ -125,6 +125,10 @@ define Build/Configure/Default
 	)
 endef
 
+define Build/Menuconfig/Default
+	+$(MAKE) ARCH=$(ARCH) -C $(PKG_BUILD_DIR) menuconfig
```

Comment:
Could you split this out into an extra commit please
So one commit adding basic bearbox support, one adding this menuconfig support and the last one adding it for ar71xx.

Reply:
OK. How do split this out into an extra commit.
Close this pull and resubmit pull?